src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1228,7 +1230,6 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 PrintFullPaths = printFullPaths,
                 ShouldIncludeErrorEndLocation = errorEndLocation,
                 PreferredUILang = preferredUILang,
-                SqmSessionGuid = sqmSessionGuid,
```

Comment:
This is still (admittedly) a public API, does anything break if we change this behavior? I can't imagine so, but it seems odd to me at first.